app/src/main/java/com/github/se/polyfit/ui/screen/settings/AccountSettingsScreen.kt

```diff
@@ -117,7 +118,13 @@ private fun AccountSettings(
           keyboardOptions = KeyboardOptions.Default.copy(keyboardType = KeyboardType.Number),
           singleLine = true,
           onValueChange = {
-            val update = it.toLongOrNull()
+            var update = it.toLongOrNull()
+
+            // If there is a Long overflow, the use the previous value
+            if (update == null && it.toBigDecimalOrNull() != null) {
+              update = height.toLongOrNull()
+            }
```

Comment:
Since this format is reused throughout this file I would suggest making it a function with a param for the previous value concerned to improve code modularity

Reply:
Great suggestion ! Will do !